core/src/main/java/org/apache/accumulo/core/clientImpl/ClientContext.java

```diff
@@ -169,7 +176,8 @@ public ClientContext(SingletonReservation reservation, ClientInfo info,
     this.serverConf = serverConf;
     timeoutSupplier = memoizeWithExpiration(
         () -> getConfiguration().getTimeInMillis(Property.GENERAL_RPC_TIMEOUT));
-    sslSupplier = memoizeWithExpiration(() -> SslConnectionParams.forClient(getConfiguration()));
+    sslSupplier = memoizeWithExpiration(() -> SslConnectionParams.forClient(getConfiguration()), 1,
+        TimeUnit.DAYS);
```

Comment:
Can the SSL configuration for a client change after the client has been created? If not, then there is no reason for this to expire.

Reply:
I don't think it can. I think this was just for lazy initialization, and nothing more.